crypto/builder_test.go

```diff
@@ -258,3 +258,39 @@ func TestBuild(t *testing.T) {
 
 	t.Run("TransferMultiSignature-Schnorr", test)
 }
+
+
+func TestBuildValidatorRegistrationWithInvalidKeyLength(t *testing.T) {
+	assert.PanicsWithValue(t, "Invalid BLS public key: invalid BLS public key length", func() {
+		BuildValidatorRegistration(
+			&Transaction{
+				Asset: &TransactionAsset{
+					Validator: &ValidatorAsset{
+						// ValidatorPublicKey: "b08058db53e2665c84a40f5152e76dd2b652125a6079130d4c315e728bcf4dd1dfb44ac26e82302331d61977d3141118",
```

Comment:
Can this be removed? If it's there for reference then possibly add additional comments to it to explain